be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/Footers/FooterModelBuilder.cs

```diff
@@ -29,6 +29,12 @@ public FooterViewModel BuildModel()
         {
             var links = new List<LinkViewModel>
             {
+                new LinkViewModel
+                {
+                    Text = this.globalLocalizer["EltapFabrics"],
```

Comment:
**suggestion:** Move it off the list and check if the url is set, beacuse ADRK does not have a website with fabrics